presidio-analyzer/presidio_analyzer/country_master.csv

```diff
@@ -0,0 +1,255 @@
+﻿English_full_name,English_short_name_using_title_case,FIFA_country_code,International_olympic_committee_country_code,ISO3166-1-Alpha-2,ISO3166-1-Alpha-3,ISO3166-1-Numeric,international licence plate country code,Country_code_top_level_domain,Currency Name,ISO4217-Alpha-3,ISO4217-Numeric,Currency Minor Unit,Capital_City,Recognized_Country_Flag,Dialing_Code,Mobile_Country_Code,Mobile_Network_Code
```

Comment:
I would suggest to put this file in a new `data` folder.

Reply:
yes. i had the same inkling , implemented .